frontend/src/lib/components/apps/components/layout/AppConditionalWrapper.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+	import { getContext } from 'svelte'
+	import { initOutput } from '../../editor/appUtils'
+	import SubGridEditor from '../../editor/SubGridEditor.svelte'
+	import type { AppViewerContext, ComponentCustomCSS, RichConfiguration } from '../../types'
+	import { concatCustomCss } from '../../utils'
+	import InitializeComponent from '../helpers/InitializeComponent.svelte'
+	import { InputValue } from '../helpers'
+
+	export let id: string
+	export let componentContainerHeight: number
+	export let customCss: ComponentCustomCSS<'containercomponent'> | undefined = undefined
+	export let render: boolean
+	export let conditions: RichConfiguration[]
+
+	const { app, focusedGrid, selectedComponent, worldStore, connectingInput, componentControl } =
+		getContext<AppViewerContext>('AppViewerContext')
+
+	const outputs = initOutput($worldStore, id, {
+		conditions: [] as boolean[],
+		selectedConditionIndex: 0
+	})
+
+	function onFocus() {
+		$focusedGrid = {
+			parentComponentId: id,
+			subGridIndex: 0
+		}
+	}
+
+	$: css = concatCustomCss($app.css?.containercomponent, customCss)
+
+	let resolvedConditions: boolean[] = []
+	let selectedConditionIndex = 0
+	let forcedIndex: number = -1
+
+	function handleResolvedConditions() {
+		const slicedArray = resolvedConditions.slice(0, conditions.length)
+		outputs.conditions.set(slicedArray, true)
+
+		const firstTrueIndex = forcedIndex !== -1 ? forcedIndex : slicedArray.findIndex((c) => c)
```

Comment:
this would make the tab stuck as soon as a setTab is used. You can likely just remove forcedIndex and set selectedConditionIndex directly on setTab